conductor-client/src/main/kotlin/com/openlattice/organizations/mapstores/OrganizationExternalDatabaseTableMapstore.kt

```diff
@@ -15,6 +15,9 @@ import java.sql.PreparedStatement
 import java.sql.ResultSet
 import java.util.*
 
+const val NAME_INDEX = "name"
+const val SCHEMA_INDEX = "schema"
```

Comment:
1. These don't seem to be used in this class. Are they used somewhere else? If so, is this the best place to hold these?
2. Is there a reason you can't use the `NAME` and `SCHEMA` constants from `SerializationConstants`?

Reply:
ooh yeah these should be part of the map index config -- they're referenced elsewhere but adding that index will improve perf there.

SerializationConstants is used for Jackson serialization, and these are for hazelcast map indices which are basically shortcuts to the getters, so assuming those will be identical seems dangerous.